[{
	"raw": {},
	"required": [
		"Filename",
		"Size"
	],
	"package": "python-zope.app.broken",
	"source": "zope.app.broken",
	"version": "3.6.0-0ubuntu3",
	"section": "universe/zope",
	"priority": "extra",
	"architecture": "all",
	"depends": [
		"python-pkg-resources",
		"python-zodb",
		"python-zope",
		"python-zope.annotation",
		"python-zope.app",
		"python-zope.broken",
		"python-zope.interface",
		"python-zope.location",
		"python-zope.processlifetime",
		"python-zope.security",
		"python2.7",
		"python (>= 2.7.1-0ubuntu2)",
		"python (<< 2.8)"
	],
	"suggests": [
		"python-zope.testing"
	],
	"installedSize": 97,
	"maintainer": "Ubuntu Developers <ubuntu-devel-discuss@lists.ubuntu.com>",
	"description": "Zope Broken (ZODB) Object Support",
	"filename": "pool/universe/z/zope.app.broken/python-zope.app.broken_3.6.0-0ubuntu3_all.deb",
	"size": 8828,
	"md5": "91a9084e1f603ca414dcbaa2b00b414f",
	"sha1": "058d967eccf234ba0f25b973013920d0e7d95728",
	"sha256": "dae2fcb22af5662c119c5f653fce1fff5b4703ce972d6f46597b96d2ec8f680e",
	"descriptionMd5": "a89a611e230e4173326915cf270c439e"
}]